src/components/UserInfo/UserInfo.scss

```diff
@@ -0,0 +1,3 @@
+.UserInfo {
+  font-style: italic;
+}
```

Comment:
Consider using a more descriptive property name than `font-style`. For example, `font-family` or `font-weight`. 🖋️